scss/_variables.scss

```diff
@@ -1644,3 +1644,10 @@ $back-to-top-icon-width:         add(.5rem, 1px) !default;
 $back-to-top-icon-height:        subtract(1rem, 1px) !default;
 // scss-docs-end back-to-top
 // End mod
+
+// Stepped process
+
+$stepped-process-bar-height:  $spacer * 2 !default;
+$stepped-process-arrow-width: 1.0625rem !default;
+$stepped-process-margin-r:    $border-width !default;
+$stepped-process-item-max-width: 3.75rem !default;
```

Comment:
Also we try to align variables in the same group (on the last one in that case, would be good).